rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel = "1.65.0"
```

Comment:
I haven't used this before: can this be a semver range or is "1.65.0 or later" implied?